src/CSSTransitionGroupChild.js

```diff
@@ -19,10 +19,12 @@ import { addEndEventListener, removeEndEventListener } from './TransitionEvents'
 const TICK = 17;
 
 export class CSSTransitionGroupChild extends Component {
-	transition(animationType, finishCallback) {
-		let node = getComponentBase(this),
-			className = this.props.name[animationType] || this.props.name + '-' + animationType,
-			activeClassName = this.props.name[animationType + 'Active'] || className + '-active';
+	transition(animationType, finishCallback, timeout) {
+		let node = getComponentBase(this);
+
+		let className = this.props.name[animationType] || this.props.name + '-' + animationType;
```

Comment:
Not sure what it's doing myself, but I am not super familiar with how this lib is used. Mybe @myarcane can give us some context :)